server/integration/testsuite/src/test/java/org/infinispan/server/test/security/rest/RESTCertSecurityIT.java

```diff
@@ -136,13 +148,23 @@ private String keyAddressUnsecured(String key) {
                 + server.getRESTEndpoint().getContextPath() + "/default/" + key;
     }
 
+    private HttpResponse handleIOException(IOException e, int expectedCode) throws IOException {
```

Comment:
why not just void? This method can never return anything and can't be overridden